sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -20,6 +20,11 @@ import type { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { ParticipantsAddedEvent } from '@azure/communication-signaling';
 import { ParticipantsRemovedEvent } from '@azure/communication-signaling';
 import { ReadReceiptReceivedEvent } from '@azure/communication-signaling';
+import { StreamEndReason } from '@azure/communication-signaling';
```

Comment:
where is this class used for?

Reply:
It is for the client to know the detail around how a progressive chat message ended.